cypress/integration/data-import/view-all-filter.spec.js

```diff
@@ -0,0 +1,117 @@
+import DataImportViewAllPage from '../../support/fragments/data_import/dataImportViewAllPage';
+import fileManager from '../../support/utils/fileManager';
+import DateTools from '../../support/utils/dateTools';
+import { Accordion } from '../../../interactors';
+import getRandomPostfix from '../../support/utils/stringTools';
+import TopMenu from '../../support/fragments/topMenu';
+
+describe('ui-data-import: Filter the "View all" log screen', () => {
+  // Path to static file in fixtures
+  const pathToStaticFile = 'oneMarcBib.mrc';
+  // Create unique names for MARC files
+  const fileNameForFailedImport = `test${getRandomPostfix()}.mrc`;
+  const fileNameForSuccessfulImport = `test${getRandomPostfix()}.mrc`;
+  let userName;
+  let jobProfileName;
+  let userNameId;
+  let profileId;
+
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+    cy.getToken(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    // Create files dynamically with given name and content in fixtures
+    fileManager.createFile(`cypress/fixtures/${fileNameForFailedImport}`);
+    // read contents of static file in fixtures
+    cy.readFile(`cypress/fixtures/${pathToStaticFile}`).then(content => {
+    // and write its contents to the file which runs successfully and create it
+      fileManager.createFile(`cypress/fixtures/${fileNameForSuccessfulImport}`, content);
+    });
+
+    cy.visit(TopMenu.dataImportPath);
+    // Upload files
+    // runs with errors
+    cy.uploadFile(fileNameForFailedImport);
+    cy.reload();
+    // runs successfully
+    cy.uploadFile(fileNameForSuccessfulImport);
+
+    // Remove generated test files from fixtures after uploading
+    fileManager.deleteFile(`cypress/fixtures/${fileNameForSuccessfulImport}`);
+    fileManager.deleteFile(`cypress/fixtures/${fileNameForFailedImport}`);
+  });
+
+  beforeEach(() => {
+    DataImportViewAllPage.getSingleJobProfile().then(({ jobProfileInfo, runBy, userId }) => {
+      jobProfileName = jobProfileInfo.name;
+      userName = `${runBy.firstName} ${runBy.lastName}`;
+      userNameId = userId;
+      profileId = jobProfileInfo.id;
+    });
+  });
+
+  it('C11113 Filter the "View all" log screen', () => {
```

Comment:
could you mark your test as described in https://wiki.folio.org/display/DQA/Cypress#Cypress-Cypress-grepplugin ?